infrastructure/storage/tests/lmdb.rs

```diff
@@ -141,6 +141,24 @@ fn single_thread() {
             assert_eq!(check, *user);
         }
         assert_eq!(db.len().unwrap(), 1000);
+
+        // Print some stats
```

Comment:
I wont print this out here, it will not really add benefit to the unit tests, as unit test's only outcome is ok or not, its not some console output to check and view. This will also be lost in the terminal via all the other unit tests that report statuses.

Reply:
Agreed, was just a place holder for me to see what happens when I run a single test.

Reply:
Agreed